release_guidelines.md

```diff
@@ -12,6 +12,7 @@ An example `<version>` could be `2.0.0-rc.1`
 
 * Create a release branch: `release/<version>`. Use the final version rather than a release candidate in the branch name.
 * Update the version in `package.json`
+* Add the version to the hash in src/components/utils/versionMap.js
```

Comment:
Could you make the path enhanced with `? Like this: `src/components/utils/versionMap.js`

Reply:
Unless it's no longer valid, as I can see the file has been removed